packages/nextjs-mf/src/internal.ts

```diff
@@ -93,40 +93,36 @@ export const DEFAULT_SHARE_SCOPE: SharedObject = {
 };
```

Comment:
The logic for setting 'eager' and 'import' to undefined for all entries is unnecessary. It's better to set these properties only for the keys 'react', 'react-dom', 'next/router', and 'next/link'. This can improve performance by reducing unnecessary operations.